code/script.js

```diff
@@ -5,15 +5,132 @@
 alert(
   `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
 )
+const userName = prompt(`What is your name?`)
+const userNameUp = userName[0].toUpperCase() + userName.slice(1).toLowerCase()
+alert(`Hi ${userNameUp} !`)
 
 // Step 2 - Food choice
 // Your code goes here
 
 // Step 3 - Subtype choice
 // Your code goes here
 
+const orderMain = parseInt(prompt(`What type of food would you like to order? 
+Please enter the number of the food. 
+1. Pizza 
+2. Pasta 
+3. Salad`))
+
+console.log ("main order", orderMain, typeof orderMain)
+let mainSelected = ""
+let subtypeOrder = []
```

Comment:
One tip: variable names should follow a consistent naming convention throughout the code. For instance, you use both `orderMain` and `subtypeMenu` to refer to the same concept of the user's selection. Stick to one naming convention to avoid confusion 👍